api/filters.go

```diff
@@ -83,16 +100,13 @@ func (api *FilterAPI) addFilterBlueprint(w http.ResponseWriter, r *http.Request)
 	}
 
 	if submitted == "true" {
```

Comment:
Ironically this was done by me but if you run `golint ./...`, the result will want the value true to be a constant

Some other things should change such as passing context into functions should be the first parameter (check golint ./...)